CMakeLists.txt

```diff
@@ -163,6 +163,13 @@ ADD_CUSTOM_TARGET(setup_tests
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
   COMMENT "Enabling all tests ...")
 
+# disable the ability to run all tests in an in-source build:
```

Comment:
I'm surprised you start a comment with a lower-case letter :-)

Reply:
touché!